File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/CustomJwtException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;
import lombok.Getter;

@Getter
public class CustomJwtException extends BusinessException {

    private String value;

    public CustomJwtException(String value) {
        this(value, ErrorCode.TOKEN_COMMON);
    }

    public CustomJwtException(String value, ErrorCode errorCode) {
        super(value, errorCode);
        this.value = value;
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/CustomJwtException.kt
package com.dogGetDrunk.meetjyou.common.exception.business

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

open class CustomJwtException(
    val value: String,
    errorCode: ErrorCode = ErrorCode.TOKEN_COMMON
) : BusinessException(value, errorCode)
